docs/decisions/0002-events-naming.rst

```diff
@@ -0,0 +1,74 @@
+2. Open edX events naming
+=========================
+
+Status
+------
+
+Draft
+
+Context
+-------
+
+Besides a suitable location, event-type hooks need a form of
+identification when adding them to the Open edX platform. By doing this,
+recognizing which event to use, or debugging errors will be easier.
+
+This ADR has the purpose of defining the rules to be followed when
+naming an Open edX Event.
+
+Decisions
+---------
+
+The following decisions were made after receiving feedback from the Open edX
+community on this repository and on other platforms such as discuss.
+
+1. The name will be a ``string`` that follows the `type format`_ defined in
+the `OEP-41`_ with a minor change:
+
+``{Reverse DNS}.{Architecture Subdomain}.{Subject}.{Action}``
+
+Examples:
+
+* org.openedx.learning.course.enrollment.created
+* org.openedx.learning.student.registration.completed
+* org.openedx.learning.session.login.completed
+
+As can be seen, ``{Major Version}`` was removed from the name, meaning,
+events won't be versioned. Instead, the data sent by them will be:
```

Comment:
[seeking rationale] Can you provide additional context (perhaps in the `Consequences` section) on why this spec defers from OEP-41 in terms of _not_ including the `major` version number in the event name?